packages/dashboards/addon/templates/components/navi-dashboard.hbs

```diff
@@ -155,3 +157,10 @@
     </div>
   {{/if}}
 </div>
+
+{{#if dashboard.hasDirtyAttributes}}
+  <div class="navi-dashboard__save-dialogue">
+    <button onclick={{route-action "saveDashboard"}} class="btn btn-secondary navi-dashboard__save-button">Save Changes</button>
```

Comment:
Should save be a primary button?

Reply:
Not according to the mockup.

![image](https://user-images.githubusercontent.com/17508658/59289568-f6051c80-8c3b-11e9-851a-7f3ecec7f862.png)